README.md

```diff
@@ -157,7 +157,7 @@ sudo update-alternatives --install \
 
 # Usage
 
-Please refer to the [examples directory](https://bitbucket.org/ignitionrobotics/ign-physics/raw/default/examples/?at=default).
+Please refer to the [examples directory](https://github.com/ignitionrobotics/ign-physics/raw/master/examples/).
```

Comment:
this link is broken because the examples folder only exists on `ign-physics2`; we need to merge it forward to master

Reply:
I removed the usage section for `ign-physics1` because it doesn't have any examples. I think we should add it if we ever add examples to this version. This was probably here only because it's part of the template we used for all libs. Let me know what you think.

7573e5eab805a3d0bff52072733e8b073f93ac5d